src/go/rpk/go.mod

```diff
@@ -1,65 +1,65 @@
 module github.com/redpanda-data/redpanda/src/go/rpk
 
-go 1.19
+go 1.20
```

Comment:
I'll create a ticket to bump this also in our CI (ducktape and Github Actions)

Reply:
nvm, we already use 1.20 https://github.com/redpanda-data/redpanda/blob/dev/.github/workflows/rpk-build.yml#L38